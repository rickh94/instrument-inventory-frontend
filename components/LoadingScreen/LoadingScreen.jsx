import React from 'react'
import PropTypes from 'prop-types'
import { Paper, withStyles } from '@material-ui/core'
import { BulletList as LoadingList } from 'react-content-loader'

import {root} from '../../globalStyles'

const styles = {root}

const LoadingScreen = ({classes}) => {
  return (
    <React.Fragment>
      <Paper className={classes.root}>
        <LoadingList />
      </Paper>
    </React.Fragment>
  )
}

LoadingScreen.propTypes = {
  classes: PropTypes.object.isRequired
}

export default withStyles(styles)(LoadingScreen)
