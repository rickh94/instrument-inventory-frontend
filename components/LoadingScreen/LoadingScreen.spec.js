import React from 'react';
import LoadingScreen from './LoadingScreen';

describe('<LoadingScreen />', () => {
});
