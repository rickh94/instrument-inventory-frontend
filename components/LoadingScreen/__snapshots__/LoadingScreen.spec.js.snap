// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingScreen /> matches snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded WithStyles(ForwardRef(Paper))-root-1"
  >
    <svg
      aria-labelledby="Loading interface..."
      preserveAspectRatio="none"
      role="img"
      viewBox="0 0 400 130"
    >
      <title>
        Loading interface...
      </title>
      <rect
        clip-path="url(#c4mt1gqzlqv)"
        height="130"
        style="fill: url(#hzlrjy2boss);"
        width="400"
        x="0"
        y="0"
      />
      <defs>
        <clippath
          id="c4mt1gqzlqv"
        >
          <circle
            cx="10"
            cy="20"
            r="8"
          />
          <rect
            height="10"
            rx="5"
            ry="5"
            width="220"
            x="25"
            y="15"
          />
          <circle
            cx="10"
            cy="50"
            r="8"
          />
          <rect
            height="10"
            rx="5"
            ry="5"
            width="220"
            x="25"
            y="45"
          />
          <circle
            cx="10"
            cy="80"
            r="8"
          />
          <rect
            height="10"
            rx="5"
            ry="5"
            width="220"
            x="25"
            y="75"
          />
          <circle
            cx="10"
            cy="110"
            r="8"
          />
          <rect
            height="10"
            rx="5"
            ry="5"
            width="220"
            x="25"
            y="105"
          />
        </clippath>
        <lineargradient
          id="hzlrjy2boss"
        >
          <stop
            offset="0%"
            stop-color="#f0f0f0"
            stop-opacity="1"
          >
            <animate
              attributeName="offset"
              dur="2s"
              keyTimes="0; 0.25; 1"
              repeatCount="indefinite"
              values="-2; -2; 1"
            />
          </stop>
          <stop
            offset="50%"
            stop-color="#e0e0e0"
            stop-opacity="1"
          >
            <animate
              attributeName="offset"
              dur="2s"
              keyTimes="0; 0.25; 1"
              repeatCount="indefinite"
              values="-1; -1; 2"
            />
          </stop>
          <stop
            offset="100%"
            stop-color="#f0f0f0"
            stop-opacity="1"
          >
            <animate
              attributeName="offset"
              dur="2s"
              keyTimes="0; 0.25; 1"
              repeatCount="indefinite"
              values="0; 0; 3"
            />
          </stop>
        </lineargradient>
      </defs>
    </svg>
  </div>
</div>
`;
